### PR TITLE
Closes #277 | Implement dataloader for spamid_pair

### LINKED ISSUE #277: Create dataset loader for SPAMID-PAIR
Dataloader name: `spamid-pair/spamid-pair.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?spamid-pair

|     Dataset| spamid-pair |
|-------------|---|
| Description | SPAMID-PAIR is data post-comment pairs collected from 13 selected Indonesian public figures (artists) / public accounts with more than 15 million followers and categorized as famous artists. It was collected from Instagram using an online tool and Selenium. Two persons labeled all pair data as an expert in a total of 72874 data. The data contains Unicode text (UTF-8) and emojis scrapped in posts and comments without account profile information. |
| Subsets | - |
| Languages | ind |
| Tasks | Text Classification |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | https://data.mendeley.com/datasets/fj5pbdf95t/1 |
| HF URL | - |
| Paper URL | https://dx.doi.org/10.14569/IJACSA.2022.0131110 |

### PR DESCRIPTION
**Pull request by Alex-HaochenLi**
Closes #277.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/spamid_pair/spamid_pair.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+_SUPPORTED_TASKS = [Tasks.INTENT_CLASSIFICATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class SpamidPair(datasets.GeneratorBasedBuilder):
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/spamid_pair/spamid_pair.py:73]:
```suggestion
class SpamidPairDataset(datasets.GeneratorBasedBuilder):
```

**Review by ljvmiranda921 — COMMENTED** [review]
Hi @Alex-HaochenLi , thanks for this data loader PR. I just have one nit on class naming but the data loader works as expected! 👍

**Comment by Alex-HaochenLi** [comment]
Hello @ljvmiranda921, I have revised and committed according to your review. Please have a check :)

File: seacrowd/sea_datasets/spamid_pair/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from .utils.constants import Tasks
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/spamid_pair/__init__.py:1]: I encountered error when running the test
```
    from .utils.constants import Tasks
ModuleNotFoundError: No module named 'seacrowd.sea_datasets.spamid_pair.utils'
```

I think we can leave this `__init__.py` file empty

**Review by ryanignatius — CHANGES_REQUESTED** [review]

**Review by ryanignatius — APPROVED** [review]
reviewed and tested, LGTM

**Review by ljvmiranda921 — APPROVED** [review]
LGTM!

**Pull request merged**